public/radar.js

```diff
@@ -120,6 +120,7 @@ function init(h, w, data) {
 
                 radar.add(pv.Dot)
                     .event("click", function() {
+                        $('#tab-'+this.status()).click();
```

Comment:
is there some other way rather then simulating a click on that element? I would have thought something like `.tab('show');` should work - its always best to use the supplied methods rather than simulating user-induced events.